<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="txt_qty">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Created at</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Updated at</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>KERANJANG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_namaMenu">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_idKeranjang">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Qty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nama Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Keranjang</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Catatan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_catatan">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTambah">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>TAMBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRead">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>BACA</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="txt_updatedAt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yy/MM/dd</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="txt_createdAt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yy/MM/dd</string>
   </property>
  </widget>
  <widget class="QTableView" name="tb_output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
